var/spack/repos/builtin/packages/namd/package.py

```diff
@@ -0,0 +1,150 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+import platform
+import shutil
+import sys
+from spack import *
+
+
+class Namd(MakefilePackage):
+    """NAMDis a parallel molecular dynamics code designed for
+    high-performance simulation of large biomolecular systems."""
+
+    homepage = "http://www.ks.uiuc.edu/Research/namd/"
+    url      = "file://NAMD_2.12_Source.tar.gz"
```

Comment:
Does this actually work? For most other packages I've used:
```python
url = 'file://{0}/NAMD_2.12_Source.tar.gz'.format(os.getcwd())
```
This allows Spack to download the file if it is present in the current directory.